packages/ui5-app/webapp/test/e2e-wdi5/tests.test.js

```diff
@@ -0,0 +1,100 @@
+const title = {
+    wdio_ui5_key: "MainViewTitle",
+    forceSelect: true,
+    selector: {
+        viewName: "test.Sample.view.Main",
+        controlType: "sap.m.Title",
+        properties: {
+            text: "#UI5 demo"
+        }
+    }
+}
+
+const navFwdButton = {
+    selector: {
+        viewName: "test.Sample.view.Main",
+        id: "NavButton"
+    }
+}
+
+const list = {
+    selector: {
+        viewName: "test.Sample.view.Other",
+        id: "PeopleList"
+    }
+}
+
+const backButton = {
+    selector: {
+        id: "__page0-navButton"
```

Comment:
Is this really a good practise to use generated ID's? `__page0` works for now but can be changed at any time also in the framework.

Reply:
with a more recent version of wdi5, regex for ids are now possible :)